### PR TITLE
YLP-107 Create lambda edge generatation script for CloudFront

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
+/* eslint-disable lodash/prefer-lodash-typecheck */
 const fs = require('fs');
 const path = require('path');
 const webpack = require('webpack');
 const TerserPlugin = require('terser-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const SriWebpackPlugin = require('webpack-subresource-integrity');
```
Comment (ydrisR) [inline webpack.config.js:9]: What is the use of this new package?

**Review by ydrisR — APPROVED** [review]
LGTM
Just a question (for my culture) about `HtmlWebpackPlugin`

File: artifact.sh [hunk 2]
```diff
@@ -49,6 +49,9 @@ fi
 
 bash build.sh
 
+# CloudFront Lambda edge tests:
+npm run test-cloudfront
+
```
Comment (ydrisR) [inline artifact.sh:54]: Bellow (line 59) we should also prepare the folder that will be uploaded on the cloudfront S3 bucket.
Something like:  
```
rsync server/dist cloudfront/distrib --exclude "index.html, config*.js, cloudfront*.js"
cp server/dist/cloudfront*.js cloudfront/lambda
```

Then later in the publish process we can push the website content to cloudfront S3 and pulish the lambda

**Review by ydrisR — CHANGES_REQUESTED** [review]
Looks good overall 😎 
See comments

**Review by ydrisR — APPROVED** [review]
OK

File: webpack.config.js (hunk 1 above)
Comment (yvon-dblg) [reply webpack.config.js:9]: It's add SHA-512 (or others SHA versions)  to the link & script HTML elements

File: artifact.sh [hunk 3]
```diff
@@ -49,6 +49,9 @@ fi
 
 bash build.sh
 
+# CloudFront Lambda edge tests:
+npm run test-cloudfront
```
[the remark this answers is not in the document]
Comment (yvon-dblg) [reply artifact.sh:53]: The build must have been done before.

File: artifact.sh (hunk 2 above)
Comment (yvon-dblg) [reply artifact.sh:54]: Good point.
Too late, it's merged, we will do another PR.

**Pull request merged**